pillarbox-demo-shared/src/main/java/ch/srgssr/pillarbox/demo/shared/ui/theme/Colors.kt

```diff
@@ -8,6 +8,9 @@ package ch.srgssr.pillarbox.demo.shared.ui.theme
 
 import androidx.compose.ui.graphics.Color
 
+val srg_branding_color = Color(0xFFAF001E)
+val srg_branding_onColor = Color(0xFFFFFFFF)
```

Comment:
Shouldn't we not store those color inside another file, so we can easily replace this file from a theme generator?